src/libs/actions/Timing.ts

```diff
@@ -25,11 +30,11 @@ function start(eventName, shouldUseFirebase = false) {
 /**
  * End performance timing. Measure the time between event start/end in milliseconds, and push to Grafana
  *
- * @param {String} eventName - event name used as timestamp key
- * @param {String} [secondaryName] - optional secondary event name, passed to grafana
- * @param {number} [maxExecutionTime] - optional amount of time (ms) to wait before logging a warn
+ * @param eventName - event name used as timestamp key
+ * @param [secondaryName] - optional secondary event name, passed to grafana
+ * @param [maxExecutionTime] - optional amount of time (ms) to wait before logging a warn
  */
-function end(eventName, secondaryName = '', maxExecutionTime = 0) {
+function end(eventName: string, secondaryName = '', maxExecutionTime = 0) {
```

Comment:
Any reason why there are no types for `secondaryName` or `maxExecutionTime` as well?